README.md

```diff
@@ -103,6 +103,7 @@ module "postgres" {
 | nomad_datacenters | Nomad data centers | list(string) | ["dc1"] | no |
 | nomad_namespace | [Enterprise] Nomad namespace | string | "default" | no |
 | nomad_host_volume | Nomad host volume name | string | "persistence" | no |
+| consul_tags | Consul tags | list(string) | [""] | no |
```

Comment:
The description should be the same description as in the variable.

Reply:
I will update the description